frontend/src/metabase-types/api/search.ts

```diff
@@ -61,7 +61,7 @@ export interface SearchResult {
   table_schema: string | null;
   collection_authority_level: "official" | null;
   updated_at: string;
-  moderated_status: boolean | null;
+  moderated_status: string | null;
```

Comment:
I noticed that we don't actually use this as a boolean in our search API calls - the `moderated_status` values are either a string with `"verified"` or `null`